vision/google/cloud/vision/image.py

```diff
@@ -220,17 +220,18 @@ def _entity_from_response_type(feature_type, results):
 
     detected_objects = []
     feature_key = _REVERSE_TYPES[feature_type]
+    annotations = results.get(feature_key, [])
```

Comment:
Based on the code below, you should just short-circuit:

```python
feature_key = _REVERSE_TYPES[feature_type]
annotations = results.get(feature_key, ())
if not annotations:
    return []

detected_objects = []
# The code below is **guaranteed** to have non-empty annotations
```

----

Also note, use a tuple as the default: `results.get(feature_key, ())`